aexpect/remote_door.py

```diff
@@ -74,7 +74,10 @@
     logging.warning("Remote object backend (Pyro4) not found, some functionality"
                     " of the remote door will not be available")
 
-from aexpect import remote
+try:
+    from aexpect import remote
+except ImportError:
+    pass
```

Comment:
Yes, it should work, but again it'd be nice to add a comment here to explain why are we permitting this.

Reply:
Good point, I usually describe such things in commit messages but always prefer to put them in the code base where they are more visible and last longer. I just need to be reminded from time to time :smile:, thanks!